packages/netdog/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "netdog"
+version = "0.1.0"
+edition = "2021"
+publish = false
+build = "../build.rs"
+
+[lib]
+path = "../packages.rs"
+
+# RPM BuildRequires
+[build-dependencies]
+glibc = { path = "../glibc" }
```

Comment:
We prefer to add newlines at EOF of text files per posix standards. This is configurable in most editors.

I'd suggest doing an interactive re-base to re-add missing newlines.

Reply:
You got it